docs/dashboard/orders.md

```diff
@@ -152,3 +152,21 @@ Click the three dots in the top right corner of a Fulfilled order card. Select _
 ![Canceling shipment](screenshots/order-cancel.jpeg)
 
 For more information on Shipment configuration, see the [Shipping Methods](configuration/shipping.md).
+
+### How to generate an invoice
+
+To create an invoice for an order, click _generate invoice_ on the right side of the order details page.
```

Comment:
From what I see on the staging server now, the button says "Generate".

Reply:
Done